### PR TITLE
Add: add 2 new key bindings 'C' and 'X' for iedit-Normal model and document them.

### PR DESCRIPTION
**Pull request by xylinkun**
### PR Prelude

Thank you for working on SpaceVim! :)

Please complete these steps and check these boxes before filing your PR:

- [x] I have read and understood SpaceVim's [CONTRIBUTING](https://github.com/SpaceVim/SpaceVim/blob/master/CONTRIBUTING.md) document.
- [x] I have read and understood SpaceVim's [CODE_OF_CONDUCT](https://github.com/SpaceVim/SpaceVim/blob/master/CODE_OF_CONDUCT.md) document.
- [x] I understand my PR may be closed if it becomes obvious I didn't actually perform all of these steps.

1. Add 2 new key bindings 'C' and 'X' for iedit-Normal model and document them.
2. Add 1 new key binding '<Delete>' for iedit-Insert model to match its documentations.

### CODE REVIEW

**Comment by wsdjeg** [comment]
LGTM, let's merge it after release is done.

**Comment by xylinkun** [comment]
> LGTM, let's merge it after release is done.

What's LGTM mean?

**Comment by wsdjeg** [comment]
looks good to me.

File: autoload/SpaceVim/plugins/iedit.vim [hunk 1]
```diff
@@ -212,6 +212,16 @@ function! s:handle_normal(char) abort
       let s:cursor_stack[i].end = ''
     endfor
     redrawstatus!
+  elseif a:char == 67 " C
+    let s:mode = 'i'
+    let w:spacevim_iedit_mode = s:mode
+    let w:spacevim_statusline_mode = 'ii'
+    for i in range(len(s:cursor_stack))
+      let s:cursor_stack[i].begin = s:cursor_stack[i].begin
```
Comment (wsdjeg) [inline autoload/SpaceVim/plugins/iedit.vim]: this line can be deleted.

File: autoload/SpaceVim/plugins/iedit.vim [hunk 2]
```diff
@@ -355,11 +369,17 @@ function! s:handle_insert(char) abort
       let s:cursor_stack[i].cursor = ''
       let s:cursor_stack[i].end = ''
     endfor
-  elseif a:char ==# "\<bs>" || a:char ==# 8
+  elseif a:char ==# "\<bs>" || a:char ==# 8 " <Backspace>
```
Comment (wsdjeg) [inline autoload/SpaceVim/plugins/iedit.vim]: No need to add extra comment, the next line shows the key

File: docs/cn/documentation.md [hunk 3]
```diff
@@ -1675,27 +1675,29 @@ SpaceVim uses a powerful iedit mode to quick edit multiple occurrences of a symb
...
+| `<Left>` / `h`  | 左移光标，类似于一般模式下的 `h`                                                       |
+| `<Right>` / `l` | 右移光标，类似于一般模式下的 `l`                                                       |
+| `0` / `<Home>`  | 跳至当前 occurrence 的开头，类似于一般模式下的 `0`                                     |
+| `$` / `<End>`   | 跳至当前 occurrence 的结尾，类似于一般模式下的 `$`                                     |
+| `C`             | 删除所有 occurrences 中从光标位置开始到 occurrences 结尾的字符，类似于一般模式下的 `C` |
+| `D`             | 删除所有 occurrences 类似于一般模式下的 `D`                                            |
+| `s`             | 删除所有 occurrences 中光标下的字符并进入 `iedit-Insert` 模式，类似于一般模式下的 `s`  |
+| `S`             | 删除所有 occurrences 并进入 `iedit-Insert` 模式，类似于一般模式下的 `S`                |
+| `x`             | 删除所有 occurrences 中光标下的字符，类似于一般模式下的 `x`                            |
+| `X`             | 删除所有 occurrences 中光标前的字符，类似于一般模式下的 `X`                            |
```
Comment (wsdjeg) [inline docs/cn/documentation.md:1694]: the function of `X` is very like `x`, I do not think we should add it.

Comment (xylinkun) [reply docs/cn/documentation.md:1694]: I just want make the iedit-Normal mode more like the Normal mode.

File: autoload/SpaceVim/plugins/iedit.vim (hunk 1 above)
Comment (xylinkun) [reply autoload/SpaceVim/plugins/iedit.vim]: > this line can be deleted.

Deleted.

File: autoload/SpaceVim/plugins/iedit.vim (hunk 2 above)
Comment (xylinkun) [reply autoload/SpaceVim/plugins/iedit.vim]: Delete

**Pull request merged**